data/images/2023-05-24-strengthening-information-accessibility-title-card.yml

```diff
@@ -0,0 +1,12 @@
+
+  # https://s3.amazonaws.com/digitalgov/2023-05-24-strengthening-information-accessibility-title-card.png
+  # Image shortcode: {{< img src=2023-05-24-strengthening-information-accessibility-title-card >}}'
+  date     :  2023-05-15 15:35:00 -0400
+  uid      :  2023-05-24-strengthening-information-accessibility-title-card
+  width    :  1200
+  height   :  628
+  format   :  png
+  credit   :  
+  caption  :  
+  alt      : Title card graphic for the event has a grey background with multi-colored box shapes on the right side. In dark purple text, the first line says, Multilingual & Communicators Community of Practice. The section below has the title of the event in black bold text, Strengthening information accessibility for consumers with limited English proficiency. Below in dark purple text, says Date: May 24, 2023. The next line in dark purple text says, Time: 2:00 pm, Eastern. In the bottom left corner, is the blue Digital.gov logo which consists of three ribbons joined together in an hexagon to resemble the shape of a capital letter D, followed by the domain and program name in black bold text, Digital.gov.
+  
```

Comment:
We typically wrap the `alt` copy with double quotes to escape characters that can cause a build to fail.

```
alt      : "Title card graphic for the event has..."
```

Reply:
@nick-mon1 I will add that as a card; caption and alt used to have the quotes already set, but they disappeared after one of the updates